src/components/ContactSection/ContactSection.js

```diff
@@ -1,14 +1,65 @@
 import React from 'react';
+import styled from 'styled-components';
 import { FormattedMessage } from 'gatsby-plugin-intl';
+import { LinkedinSquare } from '@styled-icons/boxicons-logos/LinkedinSquare';
+import { Envelope } from '@styled-icons/boxicons-regular/Envelope';
+import { Phone } from '@styled-icons/boxicons-regular/Phone';
 
-import SectionHeader from 'styles/elements/SectionHeader';
+import { colors, breakPoints } from 'styles/base';
 import SectionWrapper from 'styles/elements/SectionWrapper';
+import SectionHeader from 'styles/elements/SectionHeader';
+import SectionItemWrapper from 'styles/elements/SectionItemWrapper';
+import SectionItemIcon from 'styles/elements/SectionItemIcon';
+
+const ContactSectionItemWrapper = styled(SectionItemWrapper)`
+  width: 80%;
+
+  @media (min-width: ${breakPoints.desktop}) {
+    width: 40%;
+  }
+`;
+
+const LinkedInLink = styled.a`
+  text-decoration: none;
```

Comment:
I would put it to global styles. Probably you want to reset those default styles.